packages/core/src/command/hl7-notification/hl7-notification-webhook-sender-direct.ts

```diff
@@ -79,6 +90,24 @@ export class Hl7NotificationWebhookSenderDirect implements Hl7NotificationWebhoo
     });
     log(`Conversion complete and patient entry added`);
 
+    const clinicalInformation = this.extractClinicalInformation(newEncounterData);
+
+    log(`Writing TCM encounter to DB...`);
+    await this.writeTcmEncounterToDb(
+      {
+        id: encounterId,
+        cxId,
+        patientId,
+        class: encounterClass.display,
+        facilityName,
+        admitTime: encounterPeriod?.start,
+        dischargeTime: encounterPeriod?.end,
+        clinicalInformation,
+      },
+      triggerEvent as "A01" | "A03"
```

Comment:
Isn't this typed?

Reply:
it wasn't type properly, added type narrowing util, addressed